tests/Unit/Options/Test_Options.cpp

```diff
@@ -449,37 +497,48 @@ void test_options_array_success() {
   CHECK(opts.get<Array>() == (std::array<int, 3>{{1, 2, 3}}));
 }
 
-// [[OutputRegex, In string:.*While parsing option Array:.At line 1 column
-// 8:.Failed to convert value to type \[int x3\]: .1, 2, 3, 4.]]
 SPECTRE_TEST_CASE("Unit.Options.Array.too_long", "[Unit][Options]") {
-  ERROR_TEST();
-  Options::Parser<tmpl::list<Array>> opts("");
-  opts.parse("Array: [1, 2, 3, 4]");
-  opts.get<Array>();
+  CHECK_THROWS_WITH(
+      []() {
+        Options::Parser<tmpl::list<Array>> opts("");
+        opts.parse("Array: [1, 2, 3, 4]");
+        opts.get<Array>();
+      }(),
+      Catch::Contains("While parsing option Array:") &&
+          Catch::Contains("At line 1 column 8:") &&
+          Catch::Contains("Failed to convert value to type [int x3]:") &&
+          Catch::Contains("1, 2, 3, 4"));
 }
 
-// [[OutputRegex, In string:.*While parsing option Array:.At line 2 column
-// 3:.Failed to convert value to type \[int x3\]:.  - 1.  - 2.  -
-// 3.  - 4]]
 SPECTRE_TEST_CASE("Unit.Options.Array.too_long.formatting", "[Unit][Options]") {
-  ERROR_TEST();
-  Options::Parser<tmpl::list<Array>> opts("");
-  opts.parse(
-      "Array:\n"
-      "  - 1\n"
-      "  - 2\n"
-      "  - 3\n"
-      "  - 4");
-  opts.get<Array>();
+  CHECK_THROWS_WITH(
+      []() {
+        Options::Parser<tmpl::list<Array>> opts("");
+        opts.parse(
+            "Array:\n"
+            "  - 1\n"
+            "  - 2\n"
+            "  - 3\n"
+            "  - 4");
+        opts.get<Array>();
+      }(),
+      Catch::Contains("While parsing option Array:") &&
+          Catch::Contains("At line 2 column 3:") &&
+          Catch::Contains("Failed to convert value to type [int x3]:") &&
+          Catch::Contains("- 1") && Catch::Contains("- 2") &&
+          Catch::Contains("- 3") && Catch::Contains("- 4"));
```

Comment:
Does catch not support newlines?  (This pattern shows up throughout this file, but this is more extreme than the ones before.)